app/serializers/user.js

```diff
@@ -5,9 +5,13 @@ export default ApplicationSerializer.extend({
   serialize(snapshot, options) {
     const json = this._super(...arguments);
     if (snapshot.id) {
-      let attributesToOmit = ['email'];
+      let attributesToOmit = [];
       if (!snapshot.adapterOptions || !snapshot.adapterOptions.includePassword) {
         attributesToOmit.push('password');
+      } else if (!snapshot.adapterOptions || !snapshot.adapterOptions.includeEmail) {
```

Comment:
You don't have to pick attributes. As long as the rest are null, the API should accept. We only ensure there is no password field. All the other fields can be there.

Just see how the profile page works. https://github.com/fossasia/open-event-frontend/pull/512 (The email was ignored there before because the API had an issue with email. now since that is fixed, it need not be ignored anymore.)